src/test/index.js

```diff
@@ -45,7 +49,12 @@ export const mockRequest = (props) => {
 /**
  * Setup the test library
  */
-export const install = () => {
+export const install = (
+  { enhancedDebugging } = {
+    enhancedDebugging: false,
+  }
+) => {
+  options.enhancedDebugging = enhancedDebugging
   originalGateway = configs.gateway
```

Comment:
Can you think if there is a way to "globally" enable `enhancedDebugging` via `configs` (`GlobalConfigs`) object? Would be very nice perhaps with

```
export const configs: GlobalConfigs = {
  experimentalEnhancedMockResponseOutputFormat: true
}